### PR TITLE
binance: update currency code when fetch fiat transactions ccxt/ccxt#16224

### PR DESCRIPTION
**Pull request by sc0Vu**
fix: ccxt/ccxt#16224

The currency code of USD is BUSD when get fiat orders. Added a map for this, in case binance use another currency for other fiat.

### CODE REVIEW

File: js/binance.js [hunk 1]
```diff
@@ -1146,6 +1146,9 @@ module.exports = class binance extends Exchange {
                     'JPY': true,
                     'NZD': true,
                 },
+                'reverseCurrencyToLeagalMoney': {
```
Comment (carlosmiei) [inline js/binance.js:1149]: we have a typo here, and the name does not seem very clear. Maybe something like `legalMoneyCurrenciesById` ?

Comment (sc0Vu) [reply js/binance.js:1149]: @carlosmiei that make more sense

**Comment by carlosmiei** [comment]
LGTM ✅

**Review by carlosmiei — APPROVED** [review]

**Comment by uiii** [comment]
Hi gyus, I don't see how this is related to my issue #16224

**Pull request merged**